frontend/src/pages/PipelineVersionList.tsx

```diff
@@ -85,17 +91,18 @@ class PipelineVersionList extends React.PureComponent<
     const columns: Column[] = [
       {
         customRenderer: this._nameCustomRenderer,
-        flex: 2,
+        flex: 1,
         label: 'Version name',
         sortKey: PipelineVersionSortKeys.NAME,
       },
+      { label: 'Description', flex: 3, customRenderer: descriptionCustomRenderer },
       { label: 'Uploaded on', flex: 1, sortKey: PipelineVersionSortKeys.CREATED_AT },
     ];
 
     const rows: Row[] = this.state.pipelineVersions.map(r => {
       const row = {
         id: r.id!,
-        otherFields: [r.name, formatDateString(r.created_at)] as any,
+        otherFields: [r.name, r.description!, formatDateString(r.created_at)] as any,
```

Comment:
description is optional, so I think this non-null assertion operator `!` will cause runtime failure when the description doesn't exist.